doc/userguide/rules/intro.rst

```diff
@@ -248,10 +251,48 @@ Now, let's say we have a rule with the following header::
 Only the traffic from the client to the server will be matched by this rule,
 as the direction specifies that we do not want to evaluate the response packet.
 
+Now,if we have a rule with the following header::
+
+    alert tcp 1.2.3.4 any <> 5.6.7.8 80
+
+Suricata will duplicate it and use the same rule with headers in both directions :
+
+    alert tcp 1.2.3.4 any -> 5.6.7.8 80
+    alert tcp 5.6.7.8 80 -> 1.2.3.4 any
+
 .. warning::
 
    There is no 'reverse' style direction, i.e. there is no ``<-``.
 
+Bidirectional rules
+~~~~~~~~~~~~~~~~~~~
+
+Here is an example of a bidirectional rule:
+
+.. container:: example-rule
+
+    alert http any any :example-rule-emphasis:`=>` 5.6.7.8 80 (msg:"matching both uri and status"; sid: 1; http.uri; content: "/download"; http.stat_code; content: "200";)
+
+It will match on flows to 5.6.7.8 and port 80.
+And it will match on a full transaction, using both the uri from the request,
+and the stat_code from the response.
+As such, it will match only when Suricata got both request and response.
+
+Bidirectional rules can use direction-ambiguous keywords, by first using
+``bidir.toclient`` or ``bidir.toserver`` keywords.
+
+.. container:: example-rule
+
+    alert http any any => 5.6.7.8 80 (msg:"matching json to server and xml to client"; sid: 1;  :example-rule-emphasis:`bidir.toserver;` http.content_type; content: "json";  :example-rule-emphasis:`bidir.toclient;` http.content_type; content: "xml";)
+
+Bidirectional rules have some limitations :
+- They are only meant to work on transactions with first a request to the server,
+and then a response to the client, and not the other way around (not tested).
+- They cannot have ``fast_pattern`` or ``prefilter`` on a keyword which is on
+the direction to client only if they do have any streaming buffers for detection
+on the other side.
+- They will refuse to load if a single directional rule is enough.
```

Comment:
When compiling the documents locally, I noticed that this won't render as a bulleted list. This should fix it:
```suggestion
Bidirectional rules have some limitations :

- They are only meant to work on transactions with first a request to the server,
  and then a response to the client, and not the other way around (not tested).
- They cannot have ``fast_pattern`` or ``prefilter`` on a keyword which is on
  the direction to client only if they do have any streaming buffers for detection
  on the other side.
- They will refuse to load if a single directional rule is enough.
```

Reply:
thanks

Reply:
Actually your proposal does not work :-/
> /__w/suricata/suricata/doc/userguide/rules/intro.rst:290:Unexpected indentation.

Reply:
hm, sorry about that, I tested locally and it worked. I possibly missed something when bringing from local changes to the github suggestion interface :/